packages/TelegramClientTests-Core.package/TCTCMessageTests.class/instance/testNotSupportedMessageTypeIdentifiedCorrectly.st

```diff
@@ -0,0 +1,10 @@
+testing
+testNotSupportedMessageTypeIdentifiedCorrectly
+
+	| notSupportedMessageEvent chat notSupportedMessage |
+
+ 	chat := TCTMMocks mockPrivateChat.
+	notSupportedMessageEvent := TCTMMocks mockAudioMessageJsonFrom: chat id.
```

Comment:
maybe we can use an unsupported Message that will always stay unsupported, so that the test is still correct once we support audio?

Reply:
implemented your suggestion